app/src/main/java/com/sunasterisk/youtubebackground/screen/fragment/fullscreen/FullScreenFrag.kt

```diff
@@ -0,0 +1,24 @@
+package com.sunasterisk.youtubebackground.screen.fragment.fullscreen
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+import com.sunasterisk.youtubebackground.R
+
+class FullScreenFrag : Fragment() {
```

Comment:
```suggestion
class FullScreenFragment : Fragment() {
```
Fragment này có chức năng gì vậy?